tests/test_lib.py

```diff
@@ -1,53 +1,59 @@
 import sys
 sys.path = [".."] + sys.path
 
+import os
 import decimal
 
-from . import TEST_DIR
-from . import ccmtest
-from ccmlib.cluster import Cluster
+from tests import TEST_DIR
```

Comment:
@Orenef11 I think this answered you question, seem you weren't the only encounter it

Reply:
It was indeed a bit weird. Is this a remainder from python 2?